packages/amplication-server/src/core/resource/resourceBtm.service.ts

```diff
@@ -176,26 +155,41 @@ export class ResourceBtmService {
   }
 
   generatePromptForBreakTheMonolith(resource: ResourceDataForBtm): string {
-    const entityIdNameMap = resource.entities.reduce((acc, entity) => {
-      acc[entity.id] = entity.name;
-      return acc;
-    });
+    const entityNameToRelatedFieldsMap = resource.entities.reduce(
+      (acc, entity) => {
+        const relationFields = entity.versions[0].fields.filter(
+          (field) => field.dataType === EnumDataType.Lookup
+        );
+
+        const relatedEntityFieldNames = relationFields.length
```

Comment:
This isn't `relatedEntityFieldNames` but more of `relatedEntities` or `relatedEntittiesNames`

Reply:
@mulygottlieb 
Have a look at the return value. It's the names, `string[]`

Reply:
or just and empty array

Reply:
![image](https://github.com/amplication/amplication/assets/39680385/a98176ee-0e39-4ca1-9f79-5c8e670f3922)

Reply:
@abrl91 yeah, names, but entity names and not field names. nevermind :)